projects/packages/classic-theme-helper/src/custom-content-types.php

```diff
@@ -34,18 +35,74 @@ function jetpack_load_custom_post_types() {
 	add_action( 'jetpack_activate_module_custom-content-types', array( '\Automattic\Jetpack\Classic_Theme_Helper\Jetpack_Testimonial', 'activation_post_type_support' ) );
 
 	add_action( 'init', array( '\Automattic\Jetpack\Classic_Theme_Helper\Nova_Restaurant', 'init' ) );
+
+	$site_id = Connection_Manager::get_site_id();
+	if ( is_wp_error( $site_id ) ) {
```

Comment:
Adding the site id check here prevents issues with multiple APIs, or rather failing tests. See the link in this Slack thread for more on those failing tests: p1738057987804199/1738056861.398469-slack-CDLH4C1UZ

I'd like to hear more though from anyone who has more knowledge of the interconnectedness of the REST APIs as well as the api-v2 tests as to how these may have been overlapping, if possible.

Reply:
This is now removed. I refactored to move the callback into it's own function, which prevented test failures. Commit 46692294842276be7a75646876cdc69085884411

Reply:
Checked in on Slack: p1738333639194869-slack-C05PV073SG3